torch/linalg/__init__.py

```diff
@@ -1588,11 +1608,16 @@
 .. seealso::
 
         :func:`torch.linalg.svd` computes the full singular value decomposition.
+        Also check that documentation for the selection of `driver`.
```

Comment:
Good pointer

Reply:
Nit. This might be unnecessary now as we are advertising this in the kwarg section?